packages/strapi-design-system/src/Select/Option.js

```diff
@@ -3,6 +3,27 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Box } from '../Box';
 import { Text } from '../Text';
+import { Row } from '../Row';
+import checkmarkIcon from '../BaseCheckbox/assets/checkmark.svg';
+
+const CheckMark = styled.div`
```

Comment:
ℹ️ I can't use the "BaseCheckbox" since it can't be a child of a "button" element for a11y resaons